src/js/databaseConnector.js

```diff
@@ -38,12 +41,16 @@ exports.sendFormToDatabase = dataToBase => {
 exports.getFormsFromDatabase = () => new Promise((resolve, reject) => {
     $.ajax({
         method: 'GET',
-        url: 'https://2gs2moc88g.execute-api.us-east-1.amazonaws.com/Webpage/-test',
+        url: `${invokeUrl}/-test`,
         headers: {
-            Authorization: Cognito.getToken()
+            Authorization: authMetod
         },
         contentType: 'application/json',
-        success: resp => resolve(resp.body),
+        success: resp => {
+            console.log(resp);
+            console.log(resp.body);
+            resolve(resp.body);
```

Comment:
W tym przypadku chyba nie zwróci wartości, chyba powinno być dodatkowo return, czy się jednak mylę @DamboySytesNet ?

Reply:
choroba, nie zapisałem...

Reply:
To mogę od razu wrzucić, niechcący nie zapisałem pliku jednego i dlatego to zostało :x

Reply:
I tak chce sie upewnić :D